js-api-doc/compile.d.ts

```diff
@@ -162,3 +248,61 @@ export function compileStringAsync(
   source: string,
   options?: StringOptions<'async'>
 ): Promise<CompileResult>;
+
+/**
```

Comment:
Even though it's also explained in the individual `Compiler` methods, I think it's also worth mentioning here that these methods return the same values as the top-level `compile()` and `compileAsync()` methods since it's easy to think they might not.